frontend/packages/git-service/src/utils/pac-strategy-detector.ts

```diff
@@ -0,0 +1,60 @@
+import { BaseService } from '../services/base-service';
+
+export const detectPacFiles = async (gitService: BaseService) => {
+  let pacFiles = [];
+  const isTektonFolderPresent = await gitService.isTektonFolderPresent();
+
+  if (isTektonFolderPresent) {
+    pacFiles.push('.tekton');
```

Comment:
Why do you return the folder here as well? Isn't it enough to return files - or just .yaml and .yml files here?